packages/react-hooks/README.md

```diff
@@ -29,8 +31,7 @@ import { useShape } from "@electric-sql/react"
 
 export default function MyComponent () {
   const { isUpToDate, data as fooData } = useShape({
-    shape: { table: `foo` },
-    baseUrl: "http://my-api.com/",
+    url: "http://my-api.com/shape/foo",
```

Comment:
Shouldn't the url be `"http://my-api.com/v1/shape/foo"` (i added `/v1`)

Reply:
no as the request is being proxied through their API (so the url path is somewhat arbitrary)